.github/workflows/update_npm_version.yml

```diff
@@ -70,6 +70,13 @@ jobs:
           git config --global user.name neuroid-developer
           npm version ${{ env.RELEASE_TYPE }}
 
+      - name: Update types version
```

Comment:
Same note as previous.

Reply:
what Kevin said in the previous comment.